content/docs/examples/wip/_index.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Work In Progress
```

Comment:
I think this page is overkill. I think it would be good enough to just move the tutorial up one level and add `(WIP)` to its title and put a warning on the page, which you already have, that it's a work in progress.